contacts/html-helper.js

```diff
@@ -0,0 +1,239 @@
+  const fullName = document.createElement(TAGS.SPAN);
+  fullName.id = person.fullNameId;
+  fullName.className = CALSSES.FULL_NAME;
+  const firstName = document.createElement(TAGS.SPAN);
+  firstName.innerHTML = person.firstName;
+  firstName.className = CALSSES.MARGIN_RIGHT;
+  fullName.appendChild(firstName);
+  const lastName = document.createElement(TAGS.SPAN);
+  lastName.innerHTML = person.lastName;
+  fullName.appendChild(lastName);
+  parent.appendChild(fullName);
+};
+
+const createTrashPersonHTML = (parent, person) => {
+  const trash = document.createElement(TAGS.IMAGE);
+  trash.id = person.trashId;
+  trash.src = IMAGES.TRASH;
+  trash.alt = ALT.TRASH;
+  trash.custom = CUSTOMS.TRASH;
+  trash.className = CALSSES.TRASH;
+  parent.appendChild(trash);
+};
+
+const addToHTMLNewPerson = (person) => {
+  const newLi = createListItemPersonHTML(getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST), person);
+  createPlusButtonPersonHTML(newLi, person);
+  createCountLikesPersonHTML(newLi, person);
+  createMinusButtonPersonHTML(newLi, person);
+  createAvatarImagePersonHTML(newLi, person);
+  createCheckboxPersonHTML(newLi, person);
+  createFullNamePersonHTML(newLi, person);
+  createTrashPersonHTML(newLi, person);
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST).appendChild(newLi);
+};
+
+const clearContactsListHTML = () => {
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST).innerHTML = EMPTY;
+};
+
+
+export const populateContactsListHTML = () => {
+  ContactsService.internalArrayContacts.forEach(element => addToHTMLNewPerson(element));
+};
+
+
+const reDrawContactsList = (person = ZERO, target = ZERO) => {
+  switch (target) {
+    case CUSTOMS.MINUS_BUTTON:
+      document.getElementById(person.likesCountId).innerHTML = person.likes;
+      document.getElementById(person.minusButtonId).disabled = isCountEqualToMin(person);
+      document.getElementById(person.plusButtonId).disabled = isCountEqualToMax(person);
+      break;
+    case CUSTOMS.PLUS_BUTTON:
+      document.getElementById(person.likesCountId).innerHTML = person.likes;
+      document.getElementById(person.minusButtonId).disabled = isCountEqualToMin(person);
+      document.getElementById(person.plusButtonId).disabled = isCountEqualToMax(person);
+      break;
+    case CUSTOMS.TRASH:
+      document.getElementById(person.listItemId).remove();
+      break;
+    default:
+      break;
+  }
+};
+
+
+export const reRenderContactsListHTML = (person = ZERO, tarsget = ZERO) => {
+  if (isZero(person) && isZero(tarsget)) {
+    clearContactsListHTML();
+    populateContactsListHTML();
+  }
+  reDrawContactsList(person, tarsget);
+};
+
+
+export const resetDeleteAllButtonValueAndState = () => {
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.DELETE_ALL_BUTTON).innerHTML = MESSAGES.DELETE_ALL;
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.DELETE_ALL_BUTTON).disabled = true;
+};
+
+
+export const setButtonDisabled = (button) => {
```

Comment:
you can merge setButtonDisabled and setButtonEnabled into a single function

setButtonStatus = (button, disabled) => {
   button.disabled = disabled;
}